jobmon_client/src/jobmon/client/workflow.py

```diff
@@ -530,7 +530,7 @@ def run(
         logger.info("Adding task metadata to database")
         # Need to wait for resume signal to be sent before resetting tasks, in case of a resume
         factory = WorkflowRunFactory(self.workflow_id)
-        if resume:
+        if not self._newly_created and resume:
```

Comment:
Isn't this a problem with newly created workflows specifically? I assume this PR addresses the bug where users can't use `resume=True` with new workflow IDs

Reply:
Never mind.... I missed the `not`